service-worker.js

```diff
@@ -0,0 +1,21 @@
+importScripts('https://unpkg.com/sw-toolbox@3.6.0/sw-toolbox.js');
+
+var VERSION = 1
+  , contentCacheOptions = {
+  name: 'Babel-Cache-' + VERSION,
+  maxAgeSeconds: 60 * 60 * 6
+}
+  , cdnCacheOptions = {
+  name: 'cdn',
+  maxAgeSeconds: 60 * 60 * 6
+};
```

Comment:
Could you please make multiple var declarations instead here?

Reply:
okay sure.